docs/source/alfacase/02_alfacase_syntax.rst

```diff
@@ -0,0 +1,216 @@
+
+    Check the :ref:`number-alfacase-schema` section for more information on the possible ways to enter a ``number``
+
+Case 4:
+    Similar to ``Case 3``, but instead  of a single value it accepts multiples values
+
+    .. rubric:: Example
+
+    .. code-block:: yaml
+
+        position:
+          values: [ 1.5, 4.5 ]
+          unit: m
+
+    Check the section :ref:`list-alfacase-schema` for more information about how to creates a ``list``
+
+Case 5:
+    In this case, the value is an ``Enum`` and one of the options must be filled, each attribute listed on :ref:`alfacase-reference-section`
+    has a link for the respective ``Enum`` to check all options, on the example, bellow is used a :class:`NodeCellType <alfasim_sdk.NodeCellType>`
+    that has the following options
+
+    .. autoclass:: alfasim_sdk.NodeCellType
+       :noindex:
+
+    .. rubric:: Example
+
+    .. code-block:: yaml
+
+        node_type: mass_source_boundary⠀
+
+
+Case 6:
+    The last case is a composition of components, the definition is informing that the value of physics must be
+    filled with the key and values defined for :class:`PhysicsDescription <alfasim_sdk.PhysicsDescription>`
+
+
+    .. rubric:: Example
+
+    .. code-block:: yaml
+
+        physics:
+          hydrodynamic_model: hydrodynamic_model_4_fields
+          simulation_regime: simulation_regime_transient
+
+
+All the definitions offer default values, this allows us to abbreviate the syntax and let |sdk| just use its defaults.
+Check the :ref:`alfacase-reference-section` section which informs all the default values of each attribute on each ``Description``.
+
+
+The next sections go deep on the syntax, showing different ways to fill some values.
+
+.. _string-alfacase-schema:
+
+String
+------
+
+.. code-block:: yaml
+   :caption: string syntax
+
+    # key: value
+    material_name: Another value goes here.
+
+    # It is possible to put quotes in a string, but it is not necessary
+    material_name: 'A string, enclosed in quotes.'
+
+    # variable: variable
+    key with spaces: value
+
+
+.. _number-alfacase-schema:
+
+Number
+-------
+
+.. code-block:: yaml
+   :caption: number syntax
+
+    # Integer
+    value: 100
+
+    # Float
+    value: 1.5
+
+    # Scientific Notation
+    value: 1e+12
+
+
+
+.. _list-alfacase-schema:
+
+List
+----
+
+``list`` is a sequence of values and on :program:`ALFAcase` list is denoted by a series of dashes (``-``)
+It is possible to define a list in a compressed inserting the value between brackets (``[`` ``]``)
+
+
+.. code-block:: yaml
+   :caption: list syntax
+
+    values:
+      - 1
+      - 2
+      - 3
+
+    # Flow style
+    values: [ 1, 2 , 3 ]
+
+
+.. _bool-alfacase-schema:
+
+Bool
+----
+
+``bool`` is case-insensitive and accepts the following options:
+
+    .. code-block:: yaml
+        :caption: boolean syntax
+
+        enable_solver_caching: True     # True
```

Comment:
I would just group up True and False values together instead of mixing them.

Reply:
Yep, good suggestion =)